main/src/ca/uwaterloo/flix/language/phase/typer/TypeConstraint.scala

```diff
@@ -31,6 +31,8 @@ sealed trait TypeConstraint {
     case TypeConstraint.Equality(_: Type.Var, Type.Pure, _) => (0, 0, 0)
     case TypeConstraint.Equality(Type.Pure, _: Type.Var, _) => (0, 0, 0)
     case TypeConstraint.Equality(tvar1: Type.Var, tvar2: Type.Var, _) if tvar1 != tvar2 => (0, 0, 0)
+    case TypeConstraint.EqJvmConstructor(_, _, _, _) => (0, 0, 0) // to check
+    case TypeConstraint.EqJvmMethod(_, _, _, _, _) => (0, 0, 0) // to check
```

Comment:
I would give these (1, 0, 0).

@mlutze Agree?

This code is likely to be rewritten later anyway.

Reply:
Technically, it would make sense to ask if tpe and tpes are all "known".

if so we give it the highest priority, i.e. 0, 0,0.

Otherwise, we give it 999, 999, 999 or whatever.

Reply:
Yes, (1, 0, 0) seems fine. We can adjust later as needed.